website/packages/docs/src/pages/shorthand.mdx

```diff
@@ -0,0 +1,154 @@
+---
+section: 50-Guides
+name: Shorthand properties
+---
+
+# Shorthand properties
+
+Shorthand properties are properties that are used to set the values of several other properties ("longhand properties"). Examples include the shorthand property `font`, which can be used to set the longhand properties `fontWeight`, `fontStyle`, `fontFamily`, and so on.
+
+We **do not** recommend mixing shorthand properties and longhand properties in the same component, as the precedence of the properties may not be what you expect.
+
+## How does Compiled handle shorthand and longhand properties?
+
+If shorthand sorting is enabled, then all shorthand properties in Compiled will come before their longhand forms. For example, `font` will come before `fontWeight` and `fontStyle`. The order your CSS properties appear in your source code is **ignored**.
```

Comment:
Is it worth calling out expansion…?  I don't see we do it on the site at all, can't find it, just the Media Query sorting page.
```suggestion
By default, Compiled expands a few common properties, like `margin` and `padding`, but this is complex and not done for all properties.

If shorthand sorting is enabled, then all shorthand properties in Compiled will come before their longhand forms. For example, `font` will come before `fontWeight` and `fontStyle`. The order your CSS properties appear in your source code is **ignored**.
```